docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+
+Supported file types include:
+
+- Asset Catalog
+- Plist
+- Storyboard
+- Xib
+
+The tool has 2 commands: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET. You can also add the xcsync tool path to your PATH environment variable to invoke the tool directly as a more seamless dotnet command.
+
+## Synopsis
+
+### xcsync-generate
+
+```dotnetcli
+dotnet build /t:xcsync-generate [-p|--project <PROJECT>]
+    [-tfm|--target-framework-moniker <FRAMEWORK>]
+    [-t|--target <TARGET_DIRECTORY>] [-f|--force [true|false]]
+    [-o|--open [true|false]]
+    [-v|--verbosity <LEVEL>]
+    [-d|--dotnet-path <.NET_SDK_PATH>]
+
+dotnet build /t:xcsync-generate -?|-h|--help
+```
+
+### xcsync-sync
+
+```dotnetcli
+dotnet build /t:xcsync-sync [-p|--project <PROJECT>]
+    [-tfm|--target-framework-moniker <FRAMEWORK>]
+    [-t|--target <TARGET_DIRECTORY>]
+    [-v|--verbosity <LEVEL>]
+    [-d|--dotnet-path <.NET_SDK_PATH>]
+
+dotnet build /t:xcsync-sync -?|h|--help
+```
+
+### Arguments
+
+- **`-p|--project <PROJECT>`**
+  The .NET project file to build. This is **required**.
+
+### Options
+
+These are optional; if not specified, the default value will be used.
+
+- **`-tfm|--target-framework-moniker <FRAMEWORK>`**
+  Invoke the tool for a specific framework. The framework must be defined in the project file. Examples: `net9.0-ios`, `net9.0-maccatalyst`. **Required** if the .NET project supports multiple tfms (for example, a standard MAUI project). If a single platform project, the default value will be the single tfm specified in the project file.
+- **`-t|--target <TARGET_DIRECTORY>`**
+  Directory in which to place the generated Xcode project. Default path is `./obj/xcode`
+- **`-f|--force [true|false]`**
+  Forces target directory to be overidden. Default value is `true`.
+- **`-o|--open [true|false]`**
+  Opens Xcode project. Default value is `false`.
+- **`-v|--verbosity <LEVEL>`**
+  Sets verbosity level of the command. Allowed values are `Detailed`, `Diagnostic`, `Minimal`, `Normal`, `Quiet`. Default value is `Normal`.
+- **`-d|--dotnet-path <.NET_SDK_PATH>`**
+  Sets the path to the .NET SDK to use. Default path is parent process if `dotnet`; otherwise, will use `dotnet` on the PATH environment variable.
+- **`-?|-h|--help`**
+  Prints out a description of how to use the command.
+
+### Examples
+
+- Generate and open a Xcode project for a MAUI project that supports .net9.0-ios
+
+  ```dotnetcli
+  dotnet build /t:xcsync-generate --project "path/to/maui.csproj" -tfm "net9.0-ios -o"
+  ```
+
+- Sync changes from a generated Xcode project in the default location (./obj/xcode) back to a .NET macOS project
+
+  ```dotnetcli
+  dotnet build /t:xcsync-sync --project "path/to/macOS.csproj"
```

Comment:
I find it confusing that you generate with a maui project but sync back to a macOS project. If the goal is to indicate that more than just a maui project is supported, I would keep the examples consistent using maui and then add a list of supported tfms.

Reply:
made the exs more maui centric! and the project option earlier discusses the allowed platforms